frontend/src/app/state/hierarchy.reducer.ts

```diff
@@ -25,9 +25,9 @@ export const HierarchyReducer = createReducer<HierarchyState>(
     initialState,
     on(HierarchyActions.createHierarchySuccess, (state, action): HierarchyState => {
         const hierarchies = [...state.Hierarchies];
-        
+
         const existingHierarchyIndex = hierarchies.findIndex(x => x.id === action.hierarchy.id);
```

Comment:
You can delete this existingHierarchy logic now. I think it only existed because we didn't have a backend.